paddle/fluid/framework/data_set.h

```diff
@@ -179,6 +187,11 @@ class DatasetImpl : public Dataset {
   virtual int64_t GetMemoryDataSize();
   virtual int64_t GetShuffleDataSize();
   virtual void MergeByInsId() {}
+  virtual void GenerateLocalTablesUnlock(int table_id, int feadim,
```

Comment:
I think we need to refactor Dataset in the next version. Current interface of base class is too messy.

Reply:
Agree, will pay attention to it